src/pytypes/onefuzztypes/models.py

```diff
@@ -516,6 +516,7 @@ class Job(BaseModel):
 
 class TaskHeartbeatEntry(BaseModel):
     task_id: UUID
+    job_id: UUID
```

Comment:
This needs to be optional, to allow parsing heartbeat messages from in-progress tasks running started prior to an upgrade.
```suggestion
    job_id: Optional[UUID]
```